tools/packaging/common/envoy_bootstrap.json

```diff
@@ -602,14 +602,14 @@
               }
             }
           },
-          "call_credentials": {
+          "call_credentials": [
```

Comment:
isn't this invalid json?

Reply:
No, test failed to unmarshal before this... 
CallCredentials is a repeated message...

Reply:
+1 shouldn't sts_service be wrapped by bracket like https://github.com/istio/istio/pull/25262/files#diff-d42383199a57207656d7209e2b0dd368R498? Does the test added exercise this template?

Reply:
Is proto Json doing some permissive parsing, if the passes as is...
I have seen this in bootstrap before.

Best to wrap the object in braces. Perhaps that is what peter meant.

Reply:
Sts_service doesn't need bracket because it's not a repeated message... and test does exercise this template

Reply:
Fixed

Reply:
Did tests pass with previous json?

Reply:
yes all the tests did...

Reply:
Can you find out why, tests should not pass with invalid config. Conversely if tests pass then config is not invalid.

Reply:
Proto JSON is not same as JSON, e.g. trailing commas are allowed: https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/util/internal/json_stream_parser.cc#L685